sound/soc/sof/trace.c

```diff
@@ -556,7 +556,6 @@ void snd_sof_release_trace(struct snd_sof_dev *sdev)
 
 	sdev->dtrace_is_enabled = false;
 	sdev->dtrace_draining = true;
-	wake_up(&sdev->trace_sleep);
 }
```

Comment:
Hmm, how can you be sure the user-space thread is not in sof_wait_trace_avail() ? The release trace can be called when sof-logger is still running and waiting for traces.